ipa-core/src/protocol/context/upgrade.rs

```diff
@@ -180,7 +183,7 @@ where
         let ctx = if self.ctx.total_records().is_specified() {
             self.ctx
         } else {
-            self.ctx.set_total_records(1)
+            self.ctx.set_total_records(typenum::Const::<1>)
```

Comment:
why not `TotalRecords::ONE`?

Reply:
No good reason... I think I did a search and replace for places that total records was called with an number.

Reply:
I think you may want to use it as people may not want to deal with typenum here